modules/machineset-troubleshooting-azure-ultra-disk.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating_machinesets/creating-machineset-azure.adoc
+// * storage/persistent_storage/persistent-storage-azure.adoc
+// * storage/persistent_storage/persistent-storage-csi-azure.adoc
+
+ifeval::["{context}" == "creating-machineset-azure"]
+:mapi:
+endif::[]
+ifeval::["{context}" == "persistent-storage-azure"]
+:pvc:
+endif::[]
+ifeval::["{context}" == "persistent-storage-csi-azure"]
+:pvc:
+endif::[]
+
+:_content-type: REFERENCE
+[id="machineset-troubleshooting-azure-ultra-disk_{context}"]
+= Troubleshooting resources for machine sets that enable ultra disks
+
+Use the information in this section to understand and recover from issues you might encounter.
+
+ifdef::pvc[]
+[id="ts-pvc-mounting-ultra_{context}"]
+== Unable to mount a persistent volume claim backed by an ultra disk
+
+If there is an issue mounting a persistent volume claim backed by an ultra disk, the pod becomes stuck in the `ContainerCreating` state and an alert is triggered.
+
+For example, if the `UltraSSDEnabled` parameter is not set on the machine that backs the node that hosts the pod, the following error message appears:
+
+[source,terminal]
+----
+StorageAccountType UltraSSD_LRS can be used only when additionalCapabilities.ultraSSDEnabled is set.
+----
+
+* To resolve this issue, describe the pod by running the following command:
++
+[source,terminal]
+----
+$ oc ~something~
+----
+#How does the user "describe the pod" to resolve this?#
+endif::pvc[]
+
+ifdef::mapi[]
+[id="ts-mapi-attach-misconfigure_{context}"]
+== Incorrect ultra disk configuration
+
+If an incorrect configuration of the `ultraSSDEnabled` parameter is specified in the machine set, the machine provisioning fails.
+
+For example, if the `ultraSSDEnabled` parameter is set to `Disabled`, but an ultra disk is specified in the `dataDisks` parameter, the following error message appears:
+
+[source,terminal]
+----
+StorageAccountType UltraSSD_LRS can be used only when additionalCapabilities.ultraSSDEnabled is set.
+----
+
+* To resolve this issue, verify that your machine set configuration is correct.
+
+[id="ts-mapi-attach-unsupported_{context}"]
+== Unsupported disk parameters
+
+If a region, availability zone, or instance size that is not compatible with ultra disks is specified in the machine set, the machine provisioning fails. Check the logs for the following error message:
+#Is this for if the user specifies an incompatible parameter, if they specify a compatible one that is not actually what they have, or both?#
```

Comment:
Need some info here
Cc @damdo

Reply:
Ultra disks are only available to users in certain circumstances, every combination of `region, availability zone, or instance size` can or can't be elegible for attaching ultra disks.
Users can check the machine-api controller logs to find out if that's the case.